src/test/java/com/microsoft/graph/functional/CustomRequestTests.java

```diff
@@ -41,4 +43,22 @@ public void testCustomGet() {
 		assertEquals(meGraphService.displayName, meOriginal.displayName);
 		assertEquals(meJson.get("displayName").getAsString(), meOriginal.displayName);
 	}
+	
+	/**
+	 * Test PUT with a custom request for both serialized and JSON content
+	 */
```

Comment:
you added two put methods, but only one test, can you please add one more test covering other function?

Reply:
Updated tests to cover both Put functions added.

Reply:
hey, ideally you should create two separate tests for two methods.